src/Components/Hist.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+const Hist = ()=>{
+   let local=[]
+   local.push(Object.values(localStorage))
+    return (
+        <div>
+            <h2>These were the numbers logged earlier today </h2>
+            {local ? local.map(item =>{
+                if(item !== undefined){
+                return (
+                <div>
+                    <h2>{`${item},`}</h2>
+                </div>
+                )}
+            }) :<p>sorry no entries</p>}
+        </div>
```

Comment:
I smell something could be refactored here. 

how about this?

```
 {local ? local.filter(item => item && <div><h2>{`${item},`}</h2></div>) : <p>sorry no entries</p>}
```

let me know how it works out for ya  😉